CRM/Contact/Form/CustomData.php

```diff
@@ -185,11 +185,11 @@ public function buildQuickForm() {
         );
         if ($isMultiple) {
           $this->assign('multiRecordDisplay', $this->_multiRecordDisplay);
-          $saveButtonName = $this->_copyValueId ? 'Save a Copy' : 'Save';
+          $saveButtonName = $this->_copyValueId ? ts('Save a Copy') : ts('Save');
           $this->addButtons(array(
               array(
                 'type' => 'upload',
-                'name' => ts('%1', array(1 => $saveButtonName)),
```

Comment:
arg indeed, thanks for spotting those. I will try to fix a few more before the merge.